lib/src/main/java/com/auth0/android/lock/views/SignUpFormView.java

```diff
@@ -111,15 +110,6 @@ private void addCustomFields(List<CustomField> customFields) {
         }
     }
 
-    private Map<String, String> getCustomFieldValues() {
```

Comment:
This was duped on another class. Kept on that one as a static public method. Btw, should that method `convertFieldsToMap` be renamed, now that is public?